packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/sharedobjects/SharedObjectRegistry.kt

```diff
@@ -1,9 +1,19 @@
 package expo.modules.kotlin.sharedobjects
 
+import expo.modules.kotlin.AppContext
 import expo.modules.kotlin.jni.JavaScriptObject
 
 @JvmInline
-value class SharedObjectId(val value: Int)
+value class SharedObjectId(val value: Int) {
+  fun toNativeObject(appContext: AppContext): SharedObject? {
+    return appContext.sharedObjectRegistry.toNativeObject(this)
+  }
+
+  fun toJavaScriptObject(appContext: AppContext): JavaScriptObject? {
+    val nativeObject = toNativeObject(appContext) ?: return null
+    return appContext.sharedObjectRegistry.toJavaScriptObjet(nativeObject)
```

Comment:
```suggestion
    return appContext.sharedObjectRegistry.toJavaScriptObject(nativeObject)
```
nit - typo (prolly needs changing in the function definition too)